SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	$nsxtConfigTypeValues | Add-Member -notepropertyname "cliPasswdMaxFailAttempts" -notepropertyvalue "Empty"
+	$nsxtConfigTypeValues | Add-Member -notepropertyname "cliPasswdMaxFailIntervalInSec" -notepropertyvalue "Empty"
+	$count = 0
+	
+    Try {
+		$checkServer = Test-Connection -ComputerName $server -Quiet -Count 1
+		if ($checkServer -eq "True") {
+			while ($checkServer -le 2) {
+				Write-LogMessage -Type INFO -Message "Attempting to connect to NSX Manager '$server'"
+				$NSXToken = Request-NsxToken -fqdn $server -username $user -password $pass 
+				if ($NSXToken) {
+					Write-LogMessage -Type INFO -Message "Connected to NSX Manager '$server'"
+					Write-LogMessage -Type INFO -Message "Retrieving password policy from NSX Manager '$server'"
+					$response = Get-NsxtManagerAuthPolicy -nsxtManagerNode $server
+					$nsxtConfigTypeValues.passwdMinimumLength = $response.minimum_password_length
+					$nsxtConfigTypeValues.apiPasswdMaxFailAttempts = $response.api_max_auth_failures
+					$nsxtConfigTypeValues.apiPasswdMaxFailIntervalInSec = $response.api_failed_auth_reset_period
+					$nsxtConfigTypeValues.apiPasswdUnlockIntervalInSec = $response.api_failed_auth_lockout_period
+					$nsxtConfigTypeValues.cliPasswdMaxFailAttempts = $response.cli_max_auth_failures
+					$nsxtConfigTypeValues.cliPasswdMaxFailIntervalInSec = $response.cli_failed_auth_lockout_period    
+
+					$nodeConfigValue = New-Object -TypeName psobject
+					$nodeConfigValue | Add-Member -notepropertyname "FQDN" -notepropertyvalue $server
+					$nodeConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "NSXMgr"
+					$nodeConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue $nsxtConfigTypeValues
+					$passwordPolicyNodes.Add($nodeConfigValue)
+					return $passwordPolicyNodes
+				} else {
+					if ($ErrorLog -match '"error_code":404') {
+						$count += 1
+						Write-LogMessage -Type WARNING -Message "Unable to submit request. Retry again in 20 secs (Try $count/3)"
+						Start-Sleep -s 20
+					} else {
+						Write-LogMessage -Type INFO -Error "Failed to obtain access token from NSX Manager '$server',  please verify your credentials and try again"  -ErrorAction Stop
+					}
+				}
+			}
+			Write-LogMessage -Type INFO -Error "Testing a connection to server $server failed, please check your details and try again" -ErrorAction Stop
+		} else {
+			Write-LogMessage -Type INFO -Error "Testing a connection to server $server failed, please check your details and try again" -ErrorAction Stop
+		}
+    }
+    Catch {
+        Debug-ExceptionWriter -object $_
+    }
+	Finally {
+		Write-LogMessage -Type INFO -Message "Retrieving NSX Manager '$server' password policies completed"
+	}
+}
+
+Function GetNsxtEdgeNode {
+	<#
+        Retrieving NSX Edge Node details from Transport Node ID
+    #>
+	Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$transportNodeID
+    )
```

Comment:
Add blank line after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175